Automattic/wp-calypso
# Add icons for new Jetpack products

### Changes proposed in this Pull Request

This PR adds the icons of the new Jetpack products to the Calypso codebase.

### Implementation notes

I appended `v2` to all new slugs and file names for now to identify the new ones easily.

### Testing instructions

- Download the PR
- Go to `devdocs/design/product-icon`
- Check that you see the new icons (see screenshot below)

### Screenshots

#### Devdocs
<img width="705" alt="Screen Shot 2020-08-05 at 11 41 20 AM" src="https://user-images.githubusercontent.com/1620183/89433754-af606480-d710-11ea-9b71-fd6151cdda3e.png">


#### Mockups
<img width="436" alt="Screen Shot 2020-08-05 at 11 41 37 AM" src="https://user-images.githubusercontent.com/1620183/89433762-b25b5500-d710-11ea-95a6-0d1aebcfb05e.png">
